src/vs/workbench/parts/extensions/browser/extensionsList.ts

```diff
@@ -147,6 +148,7 @@ export class Renderer implements IPagedRenderer<IExtension, ITemplateData> {
 			const isExtensionRunning = enabledExtensions.some(e => areSameExtensions(e, extension));
 
 			toggleClass(data.element, 'disabled', isInstalled && !isExtensionRunning);
+			toggleClass(data.root, 'disabled', isInstalled && !isExtensionRunning);
```

Comment:
`root` is the parent of `element`. If are adding the class `disabled` on the root, then why have it again on the child?
We will have to update the css, but lets try and see if we can add the class `disabled` on a single HTML element instead of 2